### PR TITLE
s6: two new commands for map handling

### LINKED ISSUE #607: Additional commands for vacuum
To simplify the map handling, you could probably add this lines to vacuum.py:

``` 
    @command()
    def get_recover_maps(self):
        """Get recover maps."""
        return self.send("get_recover_maps")

    @command(
        click.argument("id"),
        click.argument("map"),
    )
    def recover_map(self, id: int, map: int):
        """Set recover maps ."""
        payload = [int(id), int(map)]
        click.echo("Setting the map %s as active" % payload)
        return self.send("recover_map", payload)[0]
```

is working fine with my Roborock S6

please se PR: [https://github.com/rytilahti/python-miio/pull/608](url)

### PR DESCRIPTION
**Pull request by glompfine**
This two commands are working fine with my Roborock s6

Roborock s6 has three saved maps. Command `get_recover_maps` shows you the map id's.
With the command `recover_map` you can set one of the tree maps as active.

Use case:
You manage one map with some no-go-zones and another map without no-go-zones.
Depending on whether the apartment is prepared (chars are on the table, carpets are removed) or not, use one or the other map.

Closes #607

### CODE REVIEW

File: miio/vacuum.py [hunk 1]
```diff
@@ -540,6 +540,21 @@ def get_room_mapping(self):
         """Retrieves a list of segments."""
         return self.send("get_room_mapping")
 
+    @command()
+    def get_recover_maps(self):
+        """Get recover maps."""
```
Comment (rytilahti) [inline miio/vacuum.py]: Maybe `get_saved_maps`?

File: miio/vacuum.py [hunk 2]
```diff
@@ -540,6 +540,21 @@ def get_room_mapping(self):
...
+        return self.send("get_recover_maps")
+
+    @command(
+        click.argument("id"),
+        click.argument("map"),
+    )
+    def recover_map(self, id: int, map: int):
+        """Set recover maps."""
```
Comment (rytilahti) [inline miio/vacuum.py]: `use_saved_map` in accordance to the above command? What are ID and map in this case? Can you post an example of what get recover maps returns?

File: miio/vacuum.py (hunk 1 above)
Comment (glompfine) [reply miio/vacuum.py]: Actually its more like recover or backup maps than real saved maps. There is the current map and two backup maps, like this:
20200102_103607229_iOS

File: miio/vacuum.py (hunk 2 above)
Comment (glompfine) [reply miio/vacuum.py]: Example:
```
miiocli vacuum --ip roboip --token robotoken get_recover_maps
Running command get_recover_maps
```
returns:
```
[[1, 1575723134], [2, 1577540948], [3, 1577807926]]
```
I named the first number (1,2,3) as id and the second (1575723134,1577540948, 1577807926) as map

So in my case the command:
```
miiocli vacuum --ip roboip --token robotoken recover_map 1 1575723134
Running command recover_map
Setting the map [1, 1575723134] as active
ok
```
sets the initial (not edited) map as active:
PNG image

```
miiocli vacuum --ip roboip --token robotoken recover_map 2 1577540948
Running command recover_map
Setting the map [2, 1577540948] as active
ok
```
sets the oldest backup map (the one from 2019.12.28 14:49, at the very bottom of the screeny, map not fully visible) as active:
PNG image 1

```
miiocli vacuum --ip roboip --token robotoken recover_map 3 1577807926
Running command recover_map
Setting the map [3, 1577807926] as active
ok
```
sets the other backup map (the one from 2019.12.31 16:58) as active:
PNG image 2


So I suggest to name it `use_recover_map` according to `get_recover_maps`?

**Pull request merged**